elasticsearch/src/main/java/org/apache/calcite/adapter/elasticsearch/ElasticsearchSchemaFactory.java

```diff
@@ -56,15 +58,30 @@ public ElasticsearchSchemaFactory() {
 
     try {
       final String coordinatesString = (String) map.get("coordinates");
-      Preconditions.checkState(coordinatesString != null,
-          "'coordinates' is missing in configuration");
 
-      final Map<String, Integer> coordinates = mapper.readValue(coordinatesString,
-          new TypeReference<Map<String, Integer>>() { });
+      final String hostsString = (String) map.get("hosts");
 
-      // create client
-      final RestClient client = connect(coordinates);
+      final Set<HttpHost> hostSet = new LinkedHashSet<>();
```

Comment:
I would rather let elastic API handle uniqueness (or not) of hosts. 
Can you declare this variable as `List<HttpHost>`  ?

Reply:
Also please rename this variable to `hosts`

Reply:
Done